src/components/organisms/PreviewConfigurationEditor/PreviewConfigurationEditor.tsx

```diff
@@ -50,6 +50,7 @@ const PreviewConfigurationEditor = () => {
   });
 
   const [name, setName] = useState<string>(() => previewConfiguration?.name || '');
+  const [showNameError, setShowNameError] = useState(false);
```

Comment:
Might be better if this is a string because we can add more messages in the future with less logic